Vizzuality/trase
# Fix/unit layers data environments

## Description

This PR adds a new env variable for the request of the Unit layer data endpoint. This data is different in each environment, therefore the env variable. Also, we want to be able to test different environments on development

![image](https://user-images.githubusercontent.com/9701591/95875032-dcb13a80-0d71-11eb-87d5-752fdc957941.png)

## Testing instructions

Go to Indonesia - Palm Oil. On Kabupaten column and with one unit layer indicator selected you should be able to see data on the map tooltip. There wasn't any before.
